simapp/test_helpers.go

```diff
@@ -104,6 +144,35 @@ func AddTestAddrs(app *SimApp, ctx sdk.Context, accNum int, accAmt sdk.Int) []sd
 	return testAddrs
 }
 
+func ConvertAddrsToValAddrs(addrs []sdk.AccAddress) (valAddrs []sdk.ValAddress) {
```

Comment:
1. Missing godoc.
2. Use a preallocated array instead of named return variable.